decidim-proposals/spec/helpers/proposal_votes_helper_spec.rb

```diff
@@ -39,33 +39,8 @@ module Proposals
       end
 
       describe "#vote_limit_enabled?" do
-        context "when the current_user is not present" do
```

Comment:
Why remove this?

Reply:
Because the behavior for this changes. It will show the voting rules regardless of you being logged in, which makes more sense (you want to know the rules of the game before joining).